src/main/scala/com/softwaremill/mqperf/ReportResultsImpl.scala

```diff
@@ -9,9 +9,13 @@ import scala.collection.JavaConverters._
 import scala.language.implicitConversions
 import scala.util.Random
 
-class ReportResults(testConfigName: String) extends DynamoResultsTable with StrictLogging {
+trait ReportResults {
+  def report(metrics: ReceiverMetrics): Unit
+}
+
+class ReportResultsImpl(testConfigName: String) extends ReportResults with DynamoResultsTable with StrictLogging {
```

Comment:
How about naming this `DynamoReportResults` instead of `Impl`, since it's clearly the dynamo implementation of this trait.